ui/apps/akasha/src/components/app-routes.tsx

```diff
@@ -26,86 +30,112 @@ const AppRoutes: React.FC<unknown> = () => {
     _navigateToModal.current?.({ name: 'login', redirectTo });
   }, []);
 
+  const props: Pick<ErrorBoundaryProps, 'errorObj' | 'logger'> = {
```

Comment:
using the same variable name as React's convention `props` is misleading.

Reply:
I've updated the naming now